File: Projeto_8/models/usuario.py

"""Construtor para usuario"""
class Usuario:
    def __init__(self, nome, email):
        self.nome = nome
        self.email = email

    """Metodo que gera relatorio"""
    def gerar_relatorio(self):
        print(f"Nome: {self.nome}")
        print(f"Email: {self.email}")

File: Projeto_5/Projeto_5 - Alissa Garcia/src/Usuario.cpp
//Código de Alissa Garcia ADS S3

#include "Usuario.h"

//Implementação da Classe

// Construtor padrão
    Usuario::Usuario(){
        this->nome  = "NULL";
        this->email = "NULL";
        this->tipo  = "NULL";

        //Deixei tudo nulo por default
    }

// Construtor parametrizado
    Usuario::Usuario(string nome, string email, string tipo){
        this->nome  = nome;
        this->email = email;
        this->tipo  = tipo;
    }


// Metodo de impressao das informações:
    void Usuario::gerarRelatorio(){
        std::cout << "O nome do Usuário é: " << this->nome << endl;
        std::cout << "O e-mail é: " << this->email << endl;
        std::cout << "O tipo é: " << this->tipo << endl << endl;
        
    }
